subsys/spm/Kconfig

```diff
@@ -61,6 +61,20 @@ config SPM_SERVICE_RNG
 	  This service allows it to request random numbers from the SPM.
 endif
 
+# FIXME: Work-around for Thingy:91 until tooling issues have been worked out.
+config SPM_ALLOW_NON_SECURE_RESET
+	bool "Allow Non-Secure reset"
+	default y if BOARD_NRF9160_PCA20035
+	default n
```

Comment:
@ioannisg tracking the issue related to this PR on Jira there is a push for this being defaulted to `y` feel free to comment in Jira.